### PR TITLE
Addition of datasampling, qualitycontrol and readout

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: readout.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+} 
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                          \
```
Comment (dberzano) [inline readout.sh]: Use `\\` instead of `\`

File: readoutcard.sh [hunk 2]
```diff
@@ -42,12 +42,12 @@ proc ModulesHelp { } {
 set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
 module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
 # Dependencies GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION
-module load BASE/1.0                                                          \\
-            ${BOOST_VERSION:+boost/$BOOST_VERSION-$BOOST_REVISION}            \\
-            ${GCC_TOOLCHAIN_VERSION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION} \\
-            Common-O2/$COMMON_O2_VERSION-$COMMON_O2_REVISION                  \\
-            InfoLogger/$INFOLOGGER_VERSION-$INFOLOGGER_REVISION               \\
-            PDA/$PDA_VERSION-$PDA_REVISION                                    \\
+module load BASE/1.0                                                          \
```
Comment (dberzano) [inline readoutcard.sh]: Use `\\` instead of `\`

File: datasampling.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                          \
```
Comment (dberzano) [inline datasampling.sh]: Use `\\` instead of `\`

File: flpproto.sh [hunk 4]
```diff
@@ -65,14 +71,18 @@ set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
...
 # Dependencies
-module load BASE/1.0                                                                            \\
-            ${GCC_TOOLCHAIN_VERSION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION} \\
-            O2/$O2_VERSION-$O2_REVISION                                                         \\
-            Monitoring/$MONITORING_VERSION-$MONITORING_REVISION                                 \\
-            Configuration/$CONFIGURATION_VERSION-$CONFIGURATION_REVISION                        \\
-            Common-O2/$COMMON_O2_VERSION-$COMMON_O2_REVISION                                    \\
-            InfoLogger/$INFOLOGGER_VERSION-$INFOLOGGER_REVISION                                 \\
-            ReadoutCard/$READOUTCARD_VERSION-$READOUTCARD_REVISION                              \\
+module load BASE/1.0                                                      \
```
Comment (dberzano) [inline flpproto.sh]: Use `\\` instead of `\`

File: infologger.sh [hunk 5]
```diff
@@ -37,9 +37,9 @@ proc ModulesHelp { } {
 set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
 module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
 # Dependencies
-module load BASE/1.0                                                          \\
-            ${BOOST_VERSION:+boost/$BOOST_VERSION-$BOOST_REVISION}            \\
-            ${GCC_TOOLCHAIN_VERSION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION} \\
+module load BASE/1.0                                                          \
```
Comment (dberzano) [inline infologger.sh]: Use `\\` instead of `\`

File: qualitycontrol.sh [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+} 
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                          \
```
Comment (dberzano) [inline qualitycontrol.sh]: Use `\\` instead of `\`

File: datasampling.sh (hunk 3 above)
Comment (Barthelemy) [inline datasampling.sh]: I have had troubles with double backslash in the case a variable such as GCC_TOOLCHAIN_VERSION did not exist. Thus using a single backslash which will always work.

**Comment by Barthelemy** [comment]
@dberzano \ are now \\

**Pull request merged**